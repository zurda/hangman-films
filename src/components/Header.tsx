import React from 'react'
import styled from '@emotion/styled'
import { colours } from '../style/shared'

const Container = styled.header`
  width: 100%;
  background-color: ${colours.maastrichtBlue};
  color: ${colours.diamond};
  font-weight: bold;
  font-family: 'Courier New', Courier, monospace;
  padding: 16px 32px;
`

const Heading = styled.h1`
  margin: 0;
`

const Description = styled.p`
  margin: 0;
  color: ${colours.pictonBlue};
`

interface IHeaderProps {
  appName: string
  appDescription: string
}

const Header = ({ appName, appDescription }: IHeaderProps) => (
  <Container>
    <Heading>{appName}</Heading>
    <Description>{appDescription}</Description>
  </Container>
)

export default Header
