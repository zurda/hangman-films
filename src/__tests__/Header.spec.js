import React from 'react'
import { create } from 'react-test-renderer'
import Header from '../components/Header'

describe('Button component', () => {
  test('Matches the snapshot', () => {
    const header = create(
      <Header appDescription="Some app description" appName="Catchy name!" />
    )
    expect(header.toJSON()).toMatchSnapshot()
  })
})
