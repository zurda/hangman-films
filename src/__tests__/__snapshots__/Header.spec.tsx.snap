// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component Matches the snapshot 1`] = `
<header
  className="css-1y4p5dk"
>
  <h1
    className="css-r44k6v"
  >
    Catchy name!
  </h1>
  <p
    className="css-gs6jah"
  >
    Some app description
  </p>
</header>
`;
